### PR TITLE
PLAT-7518: Use AzureLinux

### PR DESCRIPTION
**Pull request by fraenkel**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -83,7 +83,7 @@ variable "node_pools" {
       node_labels = optional(map(string), {
         "dominodatalab.com/node-pool" = "default"
       })
-      node_os             = optional(string, "Linux")
+      node_os             = optional(string, "AzureLinux")
```
Comment (miguelhar) [inline variables.tf:86]: Do we need to validate that the `os_sku` are not mixed?(based on:)
```The Operator does not support a mix of some nodes running Amazon Linux 2 and others running a supported operating system in the same cluster.```

Comment (fraenkel) [reply variables.tf:86]: I would assume AKS will fail.

Comment (fraenkel) [reply variables.tf:86]: You can have different skus in a cluster.

Comment (fraenkel) [reply variables.tf:86]: We aren't using the operator... we are relying on the pre-installed bits.

**Review by miguelhar — APPROVED** [review]

**Pull request merged**